blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,217 @@
+
+	if (!elements.length) return null;
+
+	const getParentClassNames = () => {
+		if (elements && elements.length > 4) {
+			return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-5-plus`;
+		}
+		return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-4-less`;
+	};
+
+	return (
+		<Stack className={getParentClassNames()}>
+			{headline && headline.trim() !== "" && <Heading>{headline}</Heading>}
```

Comment:
We will need to wrap the block in a <HeadingSection> otherwise you'll have a h1 output here

```suggestion
			{headline && headline.trim() !== "" ? <Heading>{headline}</Heading> : null}
```